spring-cloud-sleuth-zipkin/src/main/java/org/springframework/cloud/sleuth/zipkin/ZipkinSpanListener.java

```diff
@@ -59,6 +59,7 @@ public ZipkinSpanListener(SpanCollector spanCollector, EndpointLocator endpointL
 	@EventListener
 	@Order(0)
 	public void start(SpanAcquiredEvent event) {
+		// Starting a span in zipkin means adding: traceId, id, parentId(optional), and timestamp
 		event.getSpan().addTimelineAnnotation("acquire");
```

Comment:
basically, in a bit, we can remove the acquire/release annotations as they are obviated by the new fields Span.timestamp and duration